runtime/third-party/emyg/emyg_dtoa.h

```diff
@@ -0,0 +1,7 @@
+// from https://github.com/miloyip/dtoa-benchmark/blob/bf1fb58/src/emyg/emyg_dtoa.h
+/* emyg_dtoa.h */
+
+void emyg_dtoa (double value, char* buffer);
+
+// appended for nballerina use
+void emyg_dtoa_non_special (double value, char* buffer);
```

Comment:
Can you explain why we need this?

Reply:
`emyg_dtoa` doesn't handle NaN, infinity and -0.0, so those checks become redundant. Using it would still give the correct result.
This just avoids extra checks.